packages/launchpad/cypress/e2e/experimental-studio.cy.ts

```diff
@@ -0,0 +1,11 @@
+describe('experimentalStudio', () => {
```

Comment:
nit: this would need to sync with @ZachJW34's PR but would it make sense for this test file to be named something like, `config-warning.cy.ts` and we can add 
- warns for experimentalStudio
- warns for baseUrl can't connect?